.github/workflows/linux.yml

```diff
@@ -1,5 +1,8 @@
 name: Tests on Linux (Ubuntu 22.04, Python 3.11)
 on:
+  schedule:
+    # - cron: '0 20/72 * * *'
+    - cron: '10 * * * *'
```

Comment:
Now it is just only for experiment to have every 10 minutes on the precommit.
If the experiment is successful, I will switch to every 3 days validation on full scope.